### ISSUE TITLE
.htaccess bad path

### ISSUE THREAD

**Issue by nderambure** [crowdsecurity/cs-wordpress-bouncer#160]
Hello !

It seems the .htaccess is not up to date.
This 2 lines :

```
Redirectmatch 403 wp-content/plugins/crowdsec/logs/
Redirectmatch 403 wp-content/plugins/crowdsec/.cache/
```

should be :

```
Redirectmatch 403 wp-content/uploads/crowdsec/logs/
Redirectmatch 403 wp-content/uploads/crowdsec/cache/
```

Note that they are correctly described in doc : 
https://github.com/crowdsecurity/cs-wordpress-bouncer/blob/main/docs/USER_GUIDE.md#apache

**Comment by julienloizelet** [comment]
Hi @nderambure,

Thanks for pointing this out.

I will fix it asap.

**Comment by julienloizelet** [comment]
A fix is included in the last `2.7.0` release.

Thanks again
